katharsis-client/src/main/java/io/katharsis/client/ResourceRepositoryStub.java

```diff
@@ -0,0 +1,31 @@
+package io.katharsis.client;
+
+import java.io.Serializable;
+import java.util.List;
+
+import io.katharsis.queryParams.QueryParams;
+import io.katharsis.repository.ResourceRepository;
+
+/**
+ * Implemented by every {@link ResourceRepository} stub.
+ */
+public interface ResourceRepositoryStub<T, ID extends Serializable> extends ResourceRepository<T, ID> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00119)